doc/source/ray-core/user-spawn-processes.rst

```diff
@@ -0,0 +1,106 @@
+Lifetimes of a User-Spawn Process
+=================================
+
+To avoid leaking user-spawned processes, Ray provides mechanisms to kill all user-spawned processes when a worker that starts it exits. This feature prevents GPU memory leaks from child processes (e.g., torch).
```

Comment:
Let's improve this a bit. 

When you spawns child processes from Ray workers, you are responsible for managing the lifetime of child processes. However, it is not always possible, especially when worker crashes and child processes are spawned from libraries (torch dataloader).

To avoid leak...

Reply:
done